### PR TITLE
Feature: configure languages from env.js

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: src/api/config.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const defaultConfig: Config = {
     incrementalOrderBook: false,
     isResizable: false,
     isDraggable: false,
+    languages: ['', '', ''],
```
Comment (josadcha) [inline src/api/config.ts]: you can add `'en' `to default languages array instead of `['', '', '']`

File: src/api/config.ts [hunk 2]
```diff
@@ -48,3 +49,4 @@ export const rangerReconnectPeriod = (): number => Cryptobase.config.rangerRecon
 export const incrementalOrderBook = (): boolean => Cryptobase.config.incrementalOrderBook || false;
 export const isResizableGrid = ():boolean => Cryptobase.config.isResizable || false;
 export const isDraggableGrid = ():boolean => Cryptobase.config.isDraggable || false;
+export const languages = Cryptobase.config.languages || [];
```
Comment (josadcha) [inline src/api/config.ts]: add check here if `Cryptobase.config.languages,length > 0`
and if it's empty return `['en']`

File: src/containers/Confirm/Identity/index.tsx [hunk 3]
```diff
@@ -159,9 +160,8 @@ class IdentityComponent extends React.Component<Props, IdentityState> {
         const onSelectNationality = value => this.selectNationality(dataNationalities[value]);
 
         /* tslint:disable */
-        countries.registerLocale(require("i18n-iso-countries/langs/en.json"));
-        countries.registerLocale(require("i18n-iso-countries/langs/ru.json"));
-        countries.registerLocale(require("i18n-iso-countries/langs/zh.json"));
+        languages.length > 0 ? languages.map((l: string) => countries.registerLocale(require(`i18n-iso-countries/langs/${l}.json`))) : countries.registerLocale(require('i18n-iso-countries/langs/en.json'));
```
Comment (josadcha) [inline src/containers/Confirm/Identity/index.tsx]: an then you can just map through langs array without ` : countries.registerLocale(require('i18n-iso-countries/langs/en.json'));`

File: src/containers/Sidebar/index.tsx [hunk 4]
```diff
@@ -178,7 +163,30 @@ class SidebarContainer extends React.Component<Props, State> {
         );
     };
 
-    // tslint:disable:jsx-no-lambda
+    public getLanguageDropdownItems = () => {
+        window.console.log(languages);
```
Comment (josadcha) [inline src/containers/Sidebar/index.tsx]: remove please

File: src/containers/Sidebar/index.tsx [hunk 5]
```diff
@@ -178,7 +163,29 @@ class SidebarContainer extends React.Component<Props, State> {
...
+                <div className="dropdown-row">
+                    <img
+                        src={this.tryRequire(l) && require(`../../assets/images/sidebar/${l}.svg`)}
+                        alt={`${l}-flag-icon`}
+                    />
+                    <span>{l.toUpperCase()}</span>
+                </div>
+            </Dropdown.Item>,
+        );
+    }
```
Comment (akhlopiachyi) [inline src/containers/Sidebar/index.tsx]: add semicolon

**Comment by pkucherenk0** [comment]
I have found out that we are limited with Trading View language pull. If added language is not supported by Trading View we will encounter a loop bug with opening new trading page in Trading View space. (You can check supported languages by typing language code in link https://**zh**.tradingview.com/chart/ - supported; https://**ua**.tradingview.com/chart/ - not supported)

We probably would want to keep basic en language in Trading View for languages that are not supported.

@vpetrusenko I will need your help to figure out why my translation is not used, I have added the translation file and added language in env.js, index.ts.

**Comment by pkucherenk0** [comment]
It is still king of complicated to add new language, removing much easier.
To keep in mind that language Alpha2 format have to be take from ISO-639-1 list. Language code that is not supported with this format in react-intl will fail.

**Pull request merged**